### ISSUE TITLE
Request support for win-x64, win-x86 & linux-x64

### ISSUE THREAD

**Issue by CraigN** [matgr1/FreeImage-dotnet-core#12]
There is an issue when a .NET Core 2.1 project is published for the generic runtime identifiers (win-x64, win-x86 and linux-x64), the FreeImage native library is not copied to the output directory.

According to the [RID graph](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) if a project is published for RID win-x64 it will not include win7-x64.

Is it possible to add versionless RIDs to the package?

**Comment by freerider7777** [comment]
Yes, it would be nice... I had to separately build the library on Centos and place it in app folder.
